src/interfaces.ts

```diff
@@ -161,6 +170,15 @@ export interface TrackMetadata {
   artwork?: string | ResourceObject
 }
 
+export interface NowPlayingMetadata {
```

Comment:
I feel like this interface should extend `TrackMetadata`, as everything is pretty much the same, except the extra `elapsedTime`.

Reply:
Yes and no.. because the type checker will believe you can pass other items that are not going to be applied. But I have fixed this by abstracting a Base.